### ISSUE TITLE
Option  to save a "character" and a token counter

### ISSUE THREAD

**Issue by iceandstorm** [oobabooga/text-generation-webui#1003]
**Description**

I play a lot around with functional "characters". This happens mostly in the context of the character tab. 

![image](https://user-images.githubusercontent.com/7199682/230900851-19ee48f7-4299-4ba2-b943-b350fb698fe6.png)

I would love to have the option to save the state as a "character", This would be very useful. It would be perfect with a filename input field (possible a selection between json or yaml) and a token counter! 

Example for what I try to-do:  https://www.reddit.com/r/Oobabooga/comments/12hg36e/i_played_around_with_zweckentfremden_positive/

**Comment by Ph0rk0z** [comment]
I think also there should be an option to turn off the example dialog. Since we separate the greeting now why not that too?

I get a lot of extra crap in a lot of ex dialag.. like multiple <starts> and then no way to save the description besides manual edits. In my own characters of course I keep it concise.... And here you are working on it live and get even more pain.

**Comment by devonfae** [comment]
This functionality is available on a separate webapp here:
https://oobabooga.github.io/character-creator.html
you can then import the resulting json in the character tab of oobabooga itself.

**Comment by iceandstorm** [comment]
That is true, and I also can do it by hand and or use the tokenizer in auto1111 or other options. 

But this is not the point. I would like to be able to make many adjustments fast and save versions and see how many tokens my current character injection will eat up without copying and pasting stuff around all the time. 

Also, the instruction-following files can not be reloaded...
